[devtools-translate] Portuguese - What's New in DevTools (Chrome 111) (#5388)
GoogleChrome/developer.chrome.com: site/pt/_partials/devtools/whats-new.md

{# $content #}

### Chrome 111 {: #chrome111 }

> * [Debugging de cores HD no painel de Estilos](/pt/blog/new-in-devtools-111/#color)

Reviewer: We have used "depuração" as a translation to debugging in previous articles ([example](https://github.com/GoogleChrome/developer.chrome.com/blob/10bdcea52ff4c78a301d5e56327f16a04fafa2e6/site/pt/blog/new-in-devtools-109/index.md?plain=1#L96)). I'm not picky on using anglicisms when the english equivalent is used by developers more frequently than the translated version, as I think is the case here, but would like to keep consistency across articles. Happy to standardise to one or the other. I think this is easy to get inconsistent between translators and I'm wondering if it is worth creating a page inside the [handbook](https://developer.chrome.com/docs/handbook/) to help with this. WDYT?

* [Melhorias na UX dos breakpoints](/pt/blog/new-in-devtools-111/#breakpoint-redesign)
* [Atalhos customizáveis no Gravador](/pt/blog/new-in-devtools-111/#recorder)

> * [Melhor syntax highlighting para Angular](/pt/blog/new-in-devtools-111/#syntax)

Reviewer: Similar to the above, we've used "realce de sintaxe" in previous articles ([example](https://github.com/GoogleChrome/developer.chrome.com/blob/10bdcea52ff4c78a301d5e56327f16a04fafa2e6/site/pt/blog/new-in-devtools-109/index.md?plain=1#L130))

{# ### Chrome 111 {: #chrome111 }

* [Debugging de cores HD no painel de Estilos](/pt/blog/new-in-devtools-111/#color)
* [Melhorias na UX dos breakpoints](/pt/blog/new-in-devtools-111/#breakpoint-redesign)
* [Atalhos customizáveis no Gravador](/pt/blog/new-in-devtools-111/#recorder)
* [Melhor syntax highlighting para Angular](/pt/blog/new-in-devtools-111/#syntax)

> * [Paineis de caches reorganizador na aba Aplicação](/pt/blog/new-in-devtools-111/#cache)

Reviewer: This reads strange. It's also translated as "Reorganize caches no painel de Aplicação" later in the article, can we use that here too?

Author: ```suggestion
* [Reorganize caches no painel de Aplicação](/pt/blog/new-in-devtools-111/#cache)
```

* [Outras novidades](/pt/blog/new-in-devtools-111/#misc)
 #}

{# ### Chrome 110 {: #chrome110 }

* [Limpeza do painel Performance ao recarregar](/pt/blog/new-in-devtools-110/#perf)
* [Novidades no Gravador](/pt/blog/new-in-devtools-110/#recorder)
* [Veja e realçe códigos do fluxo do usuário no Gravador](/pt/blog/new-in-devtools-110/#recorder-code)
* [Customize os tipos de seletores de uma gravação](/pt/blog/new-in-devtools-110/#recorder-selector)

> * [Edição de fluxo de usuário enquanto grava](/pt/blog/new-in-devtools-110/#recorder-edit)

Reviewer: "Edição de fluxo de usuário durante a gravação" would read better.

Author: ```suggestion
* [Edição de fluxo de usuário durante a gravação](/pt/blog/new-in-devtools-110/#recorder-edit)
```

* [Auto formatação de código na mesma aba](/pt/blog/new-in-devtools-110/#pretty-print)
* [Melhor syntax highlight para Vue, SCSS e mais](/pt/blog/new-in-devtools-110/#highlight)

> * [Autocomplete consistente no console](/pt/blog/new-in-devtools-110/#console)

Reviewer: "Autocomplete ergonômico e consistente no Console" - adding ergonomic, which was dropped and Console in uppercase (since it refers to the feature, not a console)

> {# ### Chrome 111 {: #chrome111 }

Reviewer 2: ```suggestion
### Chrome 111 {: #chrome111 }
```

* [Debugging de cores HD no painel de Estilos](/pt/blog/new-in-devtools-111/#color)
* [Melhorias na UX dos breakpoints](/pt/blog/new-in-devtools-111/#breakpoint-redesign)
* [Atalhos customizáveis no Gravador](/pt/blog/new-in-devtools-111/#recorder)
* [Melhor syntax highlighting para Angular](/pt/blog/new-in-devtools-111/#syntax)
* [Reorganize caches no painel de Aplicação](/pt/blog/new-in-devtools-111/#cache)
* [Outras novidades](/pt/blog/new-in-devtools-111/#misc)

>  #}

Reviewer 2: ```suggestion
```

### Chrome 110 {: #chrome110 }

* [Limpeza do painel Performance ao recarregar](/pt/blog/new-in-devtools-110/#perf)
* [Novidades no Gravador](/pt/blog/new-in-devtools-110/#recorder)
* [Veja e realçe códigos do fluxo do usuário no Gravador](/pt/blog/new-in-devtools-110/#recorder-code)
* [Customize os tipos de seletores de uma gravação](/pt/blog/new-in-devtools-110/#recorder-selector)
* [Edição de fluxo de usuário durante a gravação](/pt/blog/new-in-devtools-110/#recorder-edit)
* [Auto formatação de código na mesma aba](/pt/blog/new-in-devtools-110/#pretty-print)
* [Melhor syntax highlight para Vue, SCSS e mais](/pt/blog/new-in-devtools-110/#highlight)
* [Autocomplete ergonômico e consistente no Console](/pt/blog/new-in-devtools-110/#console)
* [Outras novidades](/pt/blog/new-in-devtools-110/#misc)

>  #}

Reviewer 2: ```suggestion
```
